.github/workflows/promoteRCtoLatest.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
   announce-promotion-to-slack:
     # Only runs on main branch
-    if: github.ref == 'refs/heads/main'
+    if: ${{ github.ref == 'refs/heads/main' }}
```

Comment:
this doesn't change anything

Reply:
I just added the `{{ }}` cuz that's how the examples in the github docs show it.